notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<p>Please add a short note here that we are using SQUAD here for convenience and tutorial purposes and that users will likely eval their domain specific datasets. </p><p>Also note what are the annotation requirements needed in a dataset, i.e which datasets fields are what you described further below with "Now that we have the <code>questions</code>, <code>generated answers</code>, <code>contexts</code> and the <code>ground truths</code>, we can begin our pipeline evaluation and compute all the supported metrics."</p><p> </p><p>It is prudent to notify users of these requirements before they proceed further. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>